meerk40t/core/elements.py

```diff
@@ -4102,14 +4102,15 @@ def element_matrix(
             if len(data) == 0:
                 channel(_("No selected elements."))
                 return
-            if ty:
-                raise SyntaxError
             try:
+                # SVG 7.15.3 defines the matrix form as:
+                # [a c  e]
+                # [b d  f]
```

Comment:
This is the correct ordering here. It's svg which does sx, kx, ky, sy, tx, ty. It's not what I was used to but this is how both wxPython matrix and svg's matrixes are supposed to work.